src/Controller/EnMarche/UserController.php

```diff
@@ -27,13 +29,30 @@ class UserController extends Controller
 
     /**
      * @Route("", name="app_user_profile")
-     * @Method("GET|POST")
+     * @Method("GET")
      */
     public function profileOverviewAction(): Response
     {
         return $this->render('user/my_account.html.twig');
     }
 
+    /**
+     * @Route("/mes-dons", name="app_user_profile_donation")
+     * @Method("GET")
+     */
+    public function profileDonationAction(): Response
+    {
+        $donationRepos = $this->getDoctrine()->getRepository(Donation::class);
+        /** @var Adherent $user */
+        $user = $this->getUser();
+
+        $donations = $donationRepos->findByEmailAddressOrderByDonationAt($user);
+
+        return $this->render('user/my_donation.html.twig', [
```

Comment:
The `$donations` variable is useless, you can pass directly the repository method : 

```
return $this->render('user/my_donation.html.twig', [
     'donations' => $donationRepository->findByEmailAddressOrderedByDonatedAt($this->getUser()),
]);
```